ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_completion_service.proto

```diff
@@ -44,21 +49,26 @@ service CommandCompletionService {
 message CompletionStreamRequest {
   // Must correspond to the ledger id reported by the Ledger Identification Service.
   // Must be a valid LedgerString (as described in ``value.proto``).
+  // ``NOT_FOUND`` will be returned in case the ledger does not exist.
```

Comment:
I would have expected these error codes to be described in the comment to the corresponding call (`rpc CompletionStream`), as they describe the return value of the call. Is there a reason why you have added them to the individual request fields?

Reply:
My thinking was that adding information about possible error code to a particular field adds extra information about which field caused the error. Also, some requests are used in more than one method, so this approach reduces duplication.
If you think that having all return codes described near a method definition is clearer, I'll change that 👍 
What do you think?

Reply:
I agree with Robert. I think the method definition is the natural place to look for the error code description.